intrahospital_api/management/commands/batch_test_load.py

```diff
@@ -0,0 +1,14 @@
+"""
+A management command that runs every hour and loads in
+lab tests of patients
+"""
+from django.core.management.base import BaseCommand
+from intrahospital_api.services.lab_tests import service
+
+
+class Command(BaseCommand):
+    help = " runs a batch load since the last successful batch run. --force\
```

Comment:
I think this help string is wrong ? (In fact I think `--force` would raise some sort of exception?)

Also, while we're looking at it, sentence case for sentences...